pkg/goGit/goGit.go

```diff
@@ -14,13 +15,28 @@ import (
 	logr "github.com/sirupsen/logrus"
 )
 
-func UpdateLocalCopies(repos []gitapi.Repository, config *config.Config, user *config.User) {
+func UpdateLocalCopies(repos []gitapi.Repository, config *config.Config, account *config.Account) {
+	i := 0
 	for _, repo := range repos {
+		if i >= 5 {
+			logr.Infof("[Git] Stopping after 5 repositories ...")
+			break
+		}
+		i++
+		if account.ValidateName && !strings.Contains(repo.FullName, account.Name) {
+			if repo.Owner.Type == "organization" {
+				logr.Infof("[Git] Skipping repository %v because it doesn't contain the organization name %v", repo.FullName, account.Name)
+			} else if repo.Owner.Type == "user" {
+				logr.Infof("[Git] Skipping repository %v because it doesn't contain the user name %v", repo.FullName, account.Name)
+			}
+			continue
```

Comment:
We dont like `else`.

```go
if repo.Owner.Type == "organization" {
	logr.Infof("[Git] Skipping repository %v because it doesn't contain the organization name %v", repo.FullName, account.Name)
	continue
}

logr.Infof("[Git] Skipping repository %v because it doesn't contain the user name %v", repo.FullName, account.Name)
```

Reply:
fixed in commit 4ac830e